client/components/common/SpellCard.tsx

```diff
@@ -339,7 +65,11 @@ const SpellCard: React.FC<IProps> = ({ spell, customSet }) => {
         border: `1px solid ${theme.border?.default}`,
       }}
     >
-      {content}
+      <SpellCardContent
```

Comment:
Created SpellCardContent to be shared between regular SpellCard and SpellVariantCard.